dvc/updater.py

```diff
@@ -164,23 +179,32 @@ def _get_linux(self):
         return package_managers.get(distro.id())
 
     def _get_darwin(self):
-        if not is_binary():
-            if __file__.startswith("/usr/local/Cellar"):
-                return "formula"
-            else:
-                return "pip"
+        if is_binary():
+            # NOTE: both pkg and cask put dvc binary into /usr/local/bin,
+            # so in order to know which method of installation was used,
+            # we need to actually call `brew cask`
+            ret = os.system("brew cask ls dvc")
```

Comment:
let's create a ticket to update this - we don't use cask anymore, DVC is part of the official brew now

Reply:
Should I create the ticket? Does it mean that detecting how `dvc` was installed in brew case is related to path checking as well?